### ISSUE TITLE
Failure attempting to perform fresh install of CocoaPods - ArgumentError - comparison of Pod::Version with Gem::Version failed

### ISSUE THREAD

**Issue by ivxiv** [CocoaPods/CocoaPods#5450]
> ℹ Please fill out this template when filing an issue.
> All lines beginning with an ℹ symbol instruct you with
> what info we expect.  
> Please remove this line and all above before submitting.
- [x ] I've read and understood the [*CONTRIBUTING guidelines and have done my best effort to follow](https://github.com/CocoaPods/CocoaPods/blob/master/CONTRIBUTING.md).
# Report

Unable to install CocoaPods on my MacBook Pro / OS X 10.11.5 / 2.2 GHz i7 / 16 GB RAM
## What did you do?

Attempting fresh install of CocoaPods

$ sudo gem install cocoapods
$ pod setup
## What did you expected to happen?

CocoaPods would install
## What happened instead?

$ pod setup
Setting up CocoaPods master repo
  $ /usr/bin/git pull --ff-only
  From https://github.com/CocoaPods/Specs
     55e5669..4324754  master     -> origin/master
  Updating 55e5669..4324754
  Fast-forward
   Specs/Hyperdrive/0.3.0/Hyperdrive.podspec.json | 39 ++++++++++++++++++++++++++
   1 file changed, 39 insertions(+)
   create mode 100644 Specs/Hyperdrive/0.3.0/Hyperdrive.podspec.json

――― MARKDOWN TEMPLATE ―――――――――――――――――――――――――――――――――――――――――――――――――――――――――――

--- Command

```
/Users/stefan/.rvm/gems/ruby-2.0.0-p195/bin/pod setup
```

--- Stack

```
   CocoaPods : 1.0.1
        Ruby : ruby 2.0.0p195 (2013-05-14 revision 40734) [x86_64-darwin12.4.0]
    RubyGems : 2.0.3
        Host : Mac OS X 10.11.5 (15F34)
       Xcode : 7.3.1 (7D1014)
         Git : git version 2.7.4 (Apple Git-66)
Ruby lib dir : /Users/stefan/.rvm/rubies/ruby-2.0.0-p195/lib
Repositories : master - https://github.com/CocoaPods/Specs.git @ 43247541e63f78fedcd35d0fc3e022369d0b4a5d
```

--- Plugins

```
cocoapods-deintegrate : 1.0.0
cocoapods-plugins     : 1.0.0
cocoapods-search      : 1.0.0
cocoapods-stats       : 1.0.0
cocoapods-trunk       : 1.0.0
cocoapods-try         : 1.0.0
```

--- Error

```
ArgumentError - comparison of Pod::Version with Gem::Version failed
/Users/stefan/.rvm/gems/ruby-2.0.0-p195/gems/cocoapods-1.0.1/lib/cocoapods/sources_manager.rb:134:in `>'
/Users/stefan/.rvm/gems/ruby-2.0.0-p195/gems/cocoapods-1.0.1/lib/cocoapods/sources_manager.rb:134:in `verify_compatibility!'
/Users/stefan/.rvm/gems/ruby-2.0.0-p195/gems/cocoapods-1.0.1/lib/cocoapods/sources_manager.rb:90:in `block (2 levels) in update'
/Users/stefan/.rvm/gems/ruby-2.0.0-p195/gems/cocoapods-1.0.1/lib/cocoapods/user_interface.rb:63:in `section'
/Users/stefan/.rvm/gems/ruby-2.0.0-p195/gems/cocoapods-1.0.1/lib/cocoapods/sources_manager.rb:87:in `block in update'
/Users/stefan/.rvm/gems/ruby-2.0.0-p195/gems/cocoapods-1.0.1/lib/cocoapods/sources_manager.rb:86:in `each'
/Users/stefan/.rvm/gems/ruby-2.0.0-p195/gems/cocoapods-1.0.1/lib/cocoapods/sources_manager.rb:86:in `update'
/Users/stefan/.rvm/gems/ruby-2.0.0-p195/gems/cocoapods-1.0.1/lib/cocoapods/command/setup.rb:62:in `update_master_repo'
/Users/stefan/.rvm/gems/ruby-2.0.0-p195/gems/cocoapods-1.0.1/lib/cocoapods/command/setup.rb:25:in `block in run'
/Users/stefan/.rvm/gems/ruby-2.0.0-p195/gems/cocoapods-1.0.1/lib/cocoapods/user_interface.rb:63:in `section'
/Users/stefan/.rvm/gems/ruby-2.0.0-p195/gems/cocoapods-1.0.1/lib/cocoapods/command/setup.rb:21:in `run'
/Users/stefan/.rvm/gems/ruby-2.0.0-p195/gems/claide-1.0.0/lib/claide/command.rb:334:in `run'
/Users/stefan/.rvm/gems/ruby-2.0.0-p195/gems/cocoapods-1.0.1/lib/cocoapods/command.rb:50:in `run'
/Users/stefan/.rvm/gems/ruby-2.0.0-p195/gems/cocoapods-1.0.1/bin/pod:55:in `<top (required)>'
/Users/stefan/.rvm/gems/ruby-2.0.0-p195/bin/pod:23:in `load'
/Users/stefan/.rvm/gems/ruby-2.0.0-p195/bin/pod:23:in `<main>'
/Users/stefan/.rvm/gems/ruby-2.0.0-p195/bin/ruby_noexec_wrapper:14:in `eval'
/Users/stefan/.rvm/gems/ruby-2.0.0-p195/bin/ruby_noexec_wrapper:14:in `<main>'
```

――― TEMPLATE END ――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――

[!] Oh no, an error occurred.
## CocoaPods Environment

$ pod env
### Stack

```
   CocoaPods : 1.0.1
        Ruby : ruby 2.0.0p195 (2013-05-14 revision 40734) [x86_64-darwin12.4.0]
    RubyGems : 2.0.3
        Host : Mac OS X 10.11.5 (15F34)
       Xcode : 7.3.1 (7D1014)
         Git : git version 2.7.4 (Apple Git-66)
Ruby lib dir : /Users/stefan/.rvm/rubies/ruby-2.0.0-p195/lib
Repositories : master - https://github.com/CocoaPods/Specs.git @ 43247541e63f78fedcd35d0fc3e022369d0b4a5d
```
### Installation Source

```
Executable Path: /Users/stefan/.rvm/gems/ruby-2.0.0-p195/bin/pod
```
### Plugins

```
cocoapods-deintegrate : 1.0.0
cocoapods-plugins     : 1.0.0
cocoapods-search      : 1.0.0
cocoapods-stats       : 1.0.0
cocoapods-trunk       : 1.0.0
cocoapods-try         : 1.0.0
```
## Project that demonstrates the issue

N/A - issue is with CocoaPods installation

**Comment by ivxiv** [comment]
Was able to eventually get the installation working by updating my installation of Ruby to latest-greatest, via: $ rvm install ruby-2.2.3

**Comment by ioveracker** [comment]
I was also able to work around this by updating from Ruby 2.0.0-p247 to 2.2.0.

**Comment by adamcjohnson** [comment]
Fix worked for me also.

**Comment by iljaiwas** [comment]
Run into the same issue. Solved by running this command on macOS 10.11:

    sudo gem update -n /usr/local/bin –system
